### PR TITLE
edit: add schema id, schema name, catalog name in TableData

### PR DESCRIPTION
**Pull request by dust1**
## Rationale
Closes https://github.com/CeresDB/ceresdb/issues/1157

To be able to get the catalog from the table

## Detailed Changes

I added the `schema id`, `schema name`, `catalog name` to `MetaEditRequest` for passing them to `TableData`, but because `MetaEdit` involves passing the protoc protocol, I don't save them here.

## Test Plan

pass

### CODE REVIEW

**Comment by Rachelint** [comment]
Actually, `schema` and `table schema` are not the same things although with a same name... This confused me at the beginning, too.`Schema` can be seemed as something like `tenant`, `database`. We don't have detailed design about `How to create schema` and `Where to store the schema` now...

This is my current idea for the issue:
+ Now `schema id` is sent to `datanode` through `CreateTableRequest` and `OpenTableRequest` , we just use it and don't need to care persistence of it currently.
+ Just simply place it into `LoadRequest` and `MetaEditRequest`, and then place it into `TableData`.

**Comment by dust1** [comment]
Ok, let me revise it

**Comment by Rachelint** [comment]
I see this pr has conflicts with `main` branch now, let me solve it?

**Comment by dust1** [comment]
Just a moment, please. I'll sort it out😂

**Comment by Rachelint** [comment]
See codes, and I found `schema_name` and `catalog_name` may be needed, too currently for the final target about expose `schema_name` and `catalog_name` through `Table` interface...
Can please help to add thems with `schema_id` together in this pr...?

**Comment by dust1** [comment]
sure!

File: analytic_engine/src/instance/create.rs [hunk 1]
```diff
@@ -32,7 +32,7 @@ use crate::{
     },
     manifest::meta_edit::{AddTableMeta, MetaEdit, MetaEditRequest, MetaUpdate},
     space::SpaceRef,
-    table::data::{TableDataRef, TableShardInfo},
+    table::data::{TableDataExtraneousInfo, TableDataRef, TableShardInfo},
```
Comment (Rachelint) [inline analytic_engine/src/instance/create.rs:35]: how about naming it TableCatalogInfo?

Comment (dust1) [reply analytic_engine/src/instance/create.rs:35]: no problem!

File: analytic_engine/src/instance/create.rs [hunk 2]
```diff
@@ -96,6 +96,11 @@ impl Instance {
             MetaEditRequest {
                 shard_info: TableShardInfo::new(request.shard_id),
                 meta_edit: MetaEdit::Update(meta_update),
+                extr_info: TableCatalogInfo {
```
Comment (Rachelint) [inline analytic_engine/src/instance/create.rs:99]: How about refactor field names together, and we merge it after that!

Comment (dust1) [reply analytic_engine/src/instance/create.rs:99]: ok!

**Review by Rachelint — APPROVED** [review]
LGTM

**Comment by Rachelint** [comment]
Thanks @dust1 .

**Pull request merged**